Rakefile

```diff
@@ -62,6 +62,7 @@ begin
       puts Dir.pwd
       sh "git checkout #{specs_branch}"
       sh 'git pull'
+      sh 'git gc --prune=now'
```

Comment:
Running this on my local copy took ~20 minutes and kept a CPU pegged at 100% usage, but only saved ~50MB. I don't think it's worth the setup time